src/assets/data/Projects.ts

```diff
@@ -19,10 +19,10 @@ export const projects: ProjectInt[] = [
       'A Discord bot built on the discord.js library with TypeScript. Includes a database connection to MongoDB, multiple API calls, and a number of available features. Documentation page built with Angular.',
   },
   {
-    title: 'BattleSnake',
+    title: 'Battlesnake',

```

Comment:
Please don't change the title
```suggestion
    title: 'BattleSnake',
```